packages/config/config/devices/0x0190/adc-t_3000.json

```diff
@@ -0,0 +1,2839 @@
+{
+	"manufacturer": "Building 36 Technologies",
+	"manufacturerId": "0x0190",
+	"label": "ADC-T 3000",
+	"description": "Alarm.com Smart Thermostat",
+	"devices": [
+		{
+			"productType": "0x0006",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "HVAC System Type",
+			"description": "Configures the type of heating system used.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal",
+					"value": 0
+				},
+				{
+					"label": "Heat Pump",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Number of Heat Stages",
+			"description": "Heat Stages 0-3 Default is 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2
+		},
+		{
+			"#": "3",
+			"label": "Number of Cool Stages",
+			"description": "Cool Stages 0-2 Default is 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2
+		},
+		{
+			"#": "4",
+			"label": "Heat Fuel Type",
+			"description": "Choose type of fuel. Reality - whether unit is boiler vs forced air.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Fossil Fuel",
+					"value": 0
+				},
+				{
+					"label": "Electric",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "5[0xffff00]",
+			"label": "Calibration Temperature",
+			"description": "Allowable range: -10 to 10 in 1 °F increments.",
+			"unit": "0.1°F",
```

Comment:
I'm not really happy with the separation into the 3 partials, especially since precision and scale are readonly.
Its confusing me, so this is definitely going to confuse users.

I propose this approach:
1. Keep the other parts (size, precision, scale) of the partial, but comment them out (`// ...`) so we can easily add them back if we ever decide to do so. Mention the reason in another comment above the first time we do this.
2. Where it isn't too much (like for `6[0xffff00]` where it would be 8 options, including "Disable"), prefer specific options with `allowManualEntry: false` instead of trying to explain.
3. For the parameters with a bigger range (like here), I'd actually mention some legal values in the description, e.g. `Allowable range: -10 to 10 in 1 °F increments. Example: -100, -90, ..., 0, 10, ... 100` (did I get this right?)

Reply:
@AlCalzone just one thing regarding this plan: there is a linter that prevents having a single partial value for any given config parameter. E.g. if I leave only the value part of the float with  the `0xffff00` mask and remove the other 3 partials, then the linter will complain that the partial parameter is unnecessary and advise to remove the mask. A few options:
1) Change the linter to only complain when the mask is not shifted, so a single partial parameter defined as `6[0xff]` would fail the linter whereas `6[0xff00]` would pass.
2) Remove the linter completely as it may be useful to ignore part of a field that isn't useful no matter if it's bitshifted or not.

I'll let you decide but I think 2) makes more sense?

Reply:
Lets do a mix of both. If the single bitmask spans the entire parameter width (e.g. `0xffff` for a 2-byte value), complain.

Reply:
Done in f31e3aec7531db506d9ede7a902694d43b3219a0